pandas/tests/strings/test_extract.py

```diff
@@ -2,6 +2,7 @@
 import re
 
 import numpy as np
+import pyarrow as pa
```

Comment:
@mroeschke thanks, moved the import. I also added the requirement to the `pypy` and `numpydev` actions yaml files but some tests are still failing, so before making any additional changes, I thought I'd ask what else might need to be changed so ci can run. Looks like the `meta.yaml` file needs it added as a requirement as well?

Reply:
Instead of adding pyarrow to the dependency files you can structure the test like

```python
def test_whatever():
    pa = pytest.importorskip("pyarrow")
    series = ...
```

So the test will be skipped if pyarrow is not installed or you will have pyarrow accessible as `pa`

Reply:
Ah, fair enough 👍 Thank you. There's still one spec failing but it's also failing in main, it's due to a Numpy deprecation warning as far as I can tell.